plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -516,6 +519,11 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
                           ("txid", trx->id())
                           ("why",std::get<fc::exception_ptr>(response)->what()));
                }
+
+               //Dump transaction trace
+               fc_dlog(_trx_trace_dump_log, "[TRX_TRACE][REJECTING tx] ${entire_trace}",
+                        ("entire_trace", fc::json::to_pretty_string(response)));
```

Comment:
If we are going to dump them, might as well dump them completely expanded via the abi_serializer. Also try-catch so this does not throw.

Reply:
Will need to add the variant type to `single_type_requires_abi_v` in abi_serializer for abi_serializer to expand correctly.

Reply:
Fixed.